oozie-to-airflow/mappers/__init__.py

```diff
@@ -20,8 +20,8 @@
     "base_mapper",
     "decision_mapper",
     "dummy_mapper",
-    "file_archive_mappers",
     "kill_mapper",
+    "mapreduce_mapper",
```

Comment:
Why does this file have content?  In our case, an empty file is sufficient

Reply:
In my opinion, this is not a good solution. This import should also work if we do not define any value.
I created an experiments. I deleted the contents of the init files....and everything is working properly.
https://github.com/GoogleCloudPlatform/cloud-composer/commits/cleanup-init

Property `__all__` in module is import, when we write libraries. This allows you to limit the available module classes on the outside.  We do not design module interfaces, so it does not matter to us.

Reply:
OK. Great! It must have been some transient problem then while I was adding the static checkers. Why don't you make a PR removing it? I think it should be a separate PR to remove it and this one should go "as is"

Reply:
I did not want to create another place for discussion.  I will create a PR when we get a consensus. if I would not be right, then a separate PR creates noise.

Reply:
You misunderstood my English Kamil :). 

I think we are in perfect understanding and consensus already but there is a (small) communication problem.

I like that we had discussion here. And I love you disagreed with me and presented good arguments. And I agreed with them and we reached consensus already. The "Why don't you ?" question was not to ask you "Why did not you create it in the first place?" it was  - please go ahead and create PR. 

That is a subtle difference in English when you use past vs. present tense which you apparently did not realise :).

Reply:
I think you misunderstood my English Kamil :). 

I think we are in perfect understanding and consensus already but there is a (small) communication problem.

I like that we had discussion here. And I love you disagreed with me and presented good arguments. And I agreed with them and we reached consensus already. The "Why don't you ?" question was not to ask you "Why did not you create it in the first place?" it was  - please go ahead and create PR. 

That is a subtle difference in English when you use past vs. present tense which you apparently did not realise :).

Reply:
Yes. I was currently busy with other activities and I did not focus enough on your message. I created a separate: https://github.com/GoogleCloudPlatform/cloud-composer/pull/109

A discussion in PR is the better parts of this work. I love her too. 
 
I work on my English. This is at the top of my personal to-do list. Studies are in a lower position than English lessons. Thank you for understanding.